reputation-oracle/src/modules/webhook/webhook-incoming.entity.ts

```diff
@@ -1,11 +1,12 @@
 import { ChainId } from '@human-protocol/sdk';
-import { Column, Entity } from 'typeorm';
+import { Column, Entity, Index } from 'typeorm';
 
 import { NS } from '../../common/constants';
 import { WebhookStatus } from '../../common/enums';
 import { BaseEntity } from '../../database/base.entity';
 
 @Entity({ schema: NS, name: 'webhook_incoming' })
+@Index(['chainId', 'escrowAddress', 'payload'], { unique: true })
```

Comment:
It cannot be unique because existing rows will have null values. I am not sure if you can do it unique only for not null values